### DISCUSSION TITLE
Declaration of lvars

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#4907] [Help] [answered]
What are the variables declaration of for instance  : A32NX_AUTOPILOT_VS_SELECTED
and A32NX_AUTOPILOT_HEADING_SELECTED

Are they (signed) integers or (signed) doubles ? Is there documentation on these?

**Comment by tracernz** [comment, marked as answer]
https://github.com/flybywiresim/a32nx/blob/master/docs/a320-simvars.md

**Comment by aguther** [comment]
> What are the variables declaration of for instance  : A32NX_AUTOPILOT_VS_SELECTED
> 
> and A32NX_AUTOPILOT_HEADING_SELECTED
> 
> 
> 
> Are they (signed) integers or (signed) doubles ? Is there documentation on these?

Please use the events if you want to manipulate them via hardware. this ensures all logic is respected.
